ExtensionManifest.md

```diff
@@ -0,0 +1,88 @@
+# Extension Manifests - Ringtail 9.5.008
+Extension manifests are files that specify all the settings needed to setup and configure UI extensions. They are intended to be versioned assets that are updated in Ringtail with each new UI extension release.
+
+## Structure
+Extension manifests are authored in JSON format. Here's an example:
+```json
+{
+    "name": "Calculator",
+    "description": "A nifty 4-function calculator widget",
+    "type": "Workspace",
+    "url": "https://calculator.example.com",
+    "namePrefix": "CALC",
+    "fields": [{
+        "id": 1,
+        "name": "Sum",
+        "type": "Number",
+        "isOneToOne": true,
+        "codingSettings": "Write",
+        "excludeFromImport": true,
+    }],
+    "statistics": [{
+        "id": 1,
+        "name": "calculations performed"
+    }]
+}
+```
+
+## Top-Level Settings
+These settings define basic features required for every UIX.
+
+- `name` <[String]> Name of the UIX displayed in the case home tab list or documents workspace.
+- `type` <[String]> Location where the UIX will appear in Ringtail. Must be one of:
+  - `Workspace` - Documents area workspace pane
+  - `Case` - Case home page
+- `url` <[String]> URL of the web app to load.
+- `configuration` <[String]> (Optional) Payload stored by Ringtail and delivered to UIX upon initialization.
+- `description` <[String]> (Optional) Description to differentiate similar UIX displayed in the UIX admin area.
+- `privateKey` <[String]> (Optional) Key used to sign a JWT for authenticating Ringtail with backend UIX systems. See [security considerations](README.md#security-considerations) for more information.
+- `namePrefix` <[String]> (Optional) Prefix used to group and differentiate UIX fields and statistics from others in Ringtail. This value is required if fields or statistics are specified. **NOTE: This value cannot be changed once set.**
+- `fields` <[Array]<[Field](#fields)>> (Optional) Fields to create in each case the UIX is assigned to.
```

Comment:
"...in each case that the UIX is assigned to." Adding a "that" makes it easier for readers to parse the sentence, especially readers who aren't native English speakers. I only marked this one -- can you please check if there are other places where it would be helpful?